waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/AlluxioMetastoreFilter.java

```diff
@@ -0,0 +1,117 @@
+package com.hotels.bdp.waggledance.mapping.model;
+
+import java.util.List;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.MetaStoreFilterHook;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.apache.hadoop.hive.metastore.api.Index;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.api.NoSuchObjectException;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.PartitionSpec;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+public class AlluxioMetastoreFilter implements MetaStoreFilterHook {
+
+  public static final String S3_PREFIX = "s3://";
```

Comment:
I have just updated this to only be used in the tests (not sure in which package to add it though) since whoever uses this will bring their own filter and in the case of the Alluxio filter it will have to be configured with the right Alluxio path  depending on the location, so what I had here is a basic version that wouldn't be of much use.

Reply:
OK, which downstream modules need this? If its only one you can just move it to the src/test folder of that module. If its multiple then we'll need to see if we create some kind of a tests-artifacts that can be depended on downstream.

Reply:
Yeah, it's in 2 places, one is in the integration tests and one is in the core module, so atm I have it in core since the integration tests depend on that one anyway.